account_analytic_parent/models/account_analytic_account.py

```diff
@@ -11,16 +11,38 @@
 
 class AccountAnalyticAccount(models.Model):
     _inherit = "account.analytic.account"
+    _parent_name = "parent_id"
+    _parent_store = True
+    _parent_order = 'name'
+    _rec_name = 'complete_name'
+    _order = 'parent_left'
+
+    parent_left = fields.Integer('Left Parent', index=1)
+    parent_right = fields.Integer('Right Parent', index=1)
+    complete_name = fields.Char(
```

Comment:
Why adding this field if we have `display_name` for the same purpose?

Reply:
You are right @pedrobaeza . I've just removed field and method.